tracer/parser.go

```diff
@@ -204,10 +207,39 @@ func (p *Parser) Finish() error {
 		go p.global.Devtools.Send(b)
 	}
 
-	ev := p.global.EventTemplate.Copy()
-	ev.Set("event_id", p.eventID.String())
-	ev.Set("http_har_entry", base64.RawStdEncoding.EncodeToString(b))
-	DefaultManager.Insert(ev.String())
+	switch strings.ToLower(os.Getenv("SUBTRACE_REFLECTOR")) {
+	case "1", "t", "true", "y", "yes":
+		b, err := proto.Marshal(&pubsub.Message{
+			Concrete: &pubsub.Message_ConcreteV1{
+				ConcreteV1: &pubsub.Message_V1{
+					Underlying: &pubsub.Message_V1_Event{
+						Event: &pubsub.Event{
+							Concrete: &pubsub.Event_ConcreteV1{
+								ConcreteV1: &pubsub.Event_V1{
+									HarEntryJson: b,
+									Tags:         p.global.Config.Tags,
+								},
+							},
+						},
+					},
+				},
+			},
+		})
+
+		if err != nil {
+			slog.Error("failed to marshal pubsub message", "err", err)
+			break
+		}
+
+		DefaultPublisher.ch <- &b
```

Comment:
use `select { case ch <- b: ... default: ... }` to ensure you drop data if there's no space to enqueue